src/components/MegaMenu/MegaMenu.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="mx-auto inline-block relative z-50 mx-0 px-0"
+    data-testId="menu-container"
+    @mouseenter="showNav = true"
+    @mouseleave="showNav = false"
+    @click="$emit('toggleMobileNavs')"
```

Comment:
~~Is this event being used anywhere? If not, where/how would it be used? Maybe we should add a note to the documentation?~~

~~And can it just emit a `click` event for consistency?~~

See my note below about being able to do this with Tailwind's screen directives. I don't think we need to emit a custom event for this. Let me know if you want to pair on it.